terraform/cluster/aws/network.tf

```diff
@@ -41,6 +41,10 @@ resource "aws_subnet" "public" {
     "kubernetes.io/cluster/${var.name}" : "shared"
     "kubernetes.io/role/elb" : ""
   })
+
+  timeouts {
```

Comment:
@jasontclark shouldn't we add documentation about it somewhere?

Reply:
@kaiomagalhaes whoops! yes, I'll add it.